app/code/Magento/InventoryAdminUi/Test/Mftf/Data/SourceExtensionAttributeData.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<entities xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="urn:magento:mftf:DataGenerator/etc/dataProfileSchema.xsd">
+    <entity name="StorePickupEnabled" type="source_extension_attribute">
+        <data key="is_pickup_location_active">true</data>
```

Comment:
Maybe these changes need to be placed in `app/code/Magento/InventoryInStorePickupAdminUi/Test/Mftf/Data/SourceExtensionAttributeData.xml` ?

Reply:
Yes